data/xml/C67.xml

```diff
@@ -69,7 +69,7 @@
 </paper>
 
  <paper id="1013">
- <title>SOME LINGUISTIC ARGUMENTS IN FAVOUR OF <fixed-case>M</fixed-case>r <fixed-case>B</fixed-case>AR-<fixed-case>H</fixed-case>ILLEL’S “NON UNIQUENESS” HYPOTHESIS</title>
+ <title>SOME LINGUISTIC ARGUMENTS IN FAVOUR OF Mr BAR-HILLEL’S “NON UNIQUENESS” HYPOTHESIS</title>
```

Comment:
There were some hand edits that are lost by rerunning protect.py...I guess it's ok....

Reply:
Let's stabilize the scripts first and then clean up the output.

Reply:
It'd be easy enough to add `Bar-Hillel` to the `truelist`. I'd like to modify `protect.py` so that `BAR-HILLEL` is protected to `<fixed-case>B</fixed-case>AR-<fixed-case>H</fixed-case>ILLEL`, but it would be a little complicated and I'm not sure it's worth it.